### PR TITLE
adds the grinch's ultimatum to lobby music

### PR DESCRIPTION
**Pull request by TheRyeGuyWhoWillNowDie**
Have I truly become a spaceman?

:cl:  
rscadd: the grinch's ultimatum 
/:cl:

### CODE REVIEW

**Review by BurgerLUA — APPROVED** [review]

File: yogstation/code/controllers/subsystem/ticker.dm [hunk 1]
```diff
@@ -63,6 +63,7 @@
 		"https://www.youtube.com/watch?v=V52QTyn6e-A",						// Alistair Lindsay - RimWorld OST - Waiting For The Sun
 		"https://www.youtube.com/watch?v=0-CLAPJ1PLs",						// Pizza Tower OST - Tunnely Shimbers
 		"https://www.youtube.com/watch?v=4JkIs37a2JE"						// Jamiroquai - Virtual Insanity
```
Comment (Moltijoe) [inline yogstation/code/controllers/subsystem/ticker.dm]:
```suggestion
		"https://www.youtube.com/watch?v=4JkIs37a2JE",						// Jamiroquai - Virtual Insanity
```

**Review by Moltijoe — DISMISSED** [review]

File: yogstation/code/controllers/subsystem/ticker.dm (hunk 1 above)
Comment (BurgerLUA) [inline yogstation/code/controllers/subsystem/ticker.dm]: you fucked up no comma holy shit holy fuck

**Comment by Moltijoe** [comment]
bro can't even do a code

**Comment by JohnFulpWillard** [comment]
This is such a terrible way of handling roundstart music:

1- It will break if the youtube video is deleted
2- It relies on YouTube as a third party
3- It can still somewhat fall under copyright? Probably?

This really should be a config so admins can add whatever ogg sounds they want in the folder and it'll appear in-game, nothing being held in code.

**Review by ToasterBiome — APPROVED** [review]

**Comment by ToasterBiome** [comment]
> This is such a terrible way of handling roundstart music:
> 
> 1- It will break if the youtube video is deleted 2- It relies on YouTube as a third party 3- It can still somewhat fall under copyright? Probably?
> 
> This really should be a config so admins can add whatever ogg sounds they want in the folder and it'll appear in-game, nothing being held in code.

Unrelated to PR

**Pull request merged**